docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: "2"
+
+services:
+  ipfs:
+    build:
+      context: ./docker
+      dockerfile: ./Dockerfile.ipfs
+    container_name: ipfs_host
+    tty: true
+    privileged: true
+    ports:
+      - "4001:4001"
+      - "8080:8080"
+      - "5001:5001"
+
+  worker:
+    build:
+      context: ./docker
+      dockerfile: ./Dockerfile.worker
+      args:
+        email: 'openmined@openmined.com'
```

Comment:
A small thing I thought about as I read through, these could be defined as environment variables with defaults (or not, which causes docker-compose to fail), instead of having them hard coded to `openmined@openmined.com` and `openmined` respectively.

It might be nice in future if we were to build a setup or install script that handled these things.

Reply:
I setup a small bash script to handle those

```
./run_grid.sh [-g <gpu>] [-m <grid-mode>] [-a <host_ip>] [-n <name>] [-e <email>]
```